onboarding/serializers.py

```diff
@@ -339,4 +340,52 @@ class Meta:
             'page',
             'answers',
         )
+#
+
+
+class SectionProgressSerializer(serializers.ModelSerializer):
+    # page = PageSerializer()
```

Comment:
commented line seems unneeded, since you only initialising variables, that are not presented in the model here

Reply:
I was thinking if pages could be joined also or not.
Performance costs were more important than extra information the front-end would get from an API.
But still, maybe we will change this :man_shrugging: